src/main/java/Duke.java

```diff
@@ -1,10 +1,106 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final String LINE = "____________________________________________________________";
+    private static final String ADD_TASK_MSG = "Got it. I've added this task: ";
+    private static final String ERROR_MSG = "Sorry, I didn't quite get what you want me to do :(";
+    private static final int MAX_TASKS = 100;
+
     public static void main(String[] args) {
+        showHelloGreeting();
+        executeResponses();
+        showByeGreeting();
+    }
+
+    private static void executeResponses() {
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[MAX_TASKS];
+        int index = 0;
+        String text;
+        text = in.nextLine();
+        while (!text.equals("bye")) {
+            System.out.println(LINE);
+            String[] words = text.split(" ");
+
+            switch (words[0]) {
+                case "list":
```

Comment:
I think the case statements are indented wrongly. It should be the same level of indentation as the switch statement. 
E.g. 
switch (condition) {
case ABC:
    statements;
    // Fallthrough
case DEF:
    statements;
    break;